src/test/java/com/github/victools/jsonschema/generator/impl/SchemaGeneratorConfigImplTest.java

```diff
@@ -13,7 +13,6 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-
 package com.github.victools.jsonschema.generator.impl;
```

Comment:
No reason to remove this empty line.
```suggestion

package com.github.victools.jsonschema.generator.impl;
```